data/olm-catalog/ssp-operator.clusterserviceversion.yaml

```diff
@@ -355,11 +355,12 @@ spec:
                 env:
                 - name: KVM_INFO_IMAGE
                 - name: VALIDATOR_IMAGE
+                  value: quay.io/kubevirt/kubevirt-template-validator:latest
```

Comment:
Is this value required here?

Reply:
I'm not sure if it needs to be stored, this file is generated by the build process. We can probably add it to `.gitignore`.
I can investigate it in another PR.